submissions/kutembaChinambu/database/init.sql

```diff
@@ -0,0 +1,63 @@
+<!-- This is a query that creates the food table.-->
+
+CREATE TABLE food 
+(
+id INT NOT NULL AUTO_INCREMENT,
+fast_food VARCHAR(100) NOT NULL,
+vegetables_and_fruits VARCHAR(100) NOT NULL,
+drinks_and_cocktails VARCHAR(100) NOT NULL,
+resturants VARCHAR(100) NOT NULL,
+PRIMARY KEY (id)
+);
+
+<!-- This is a query that creates the admin table -->
+
+CREATE TABLE admin 
+(
+id INT NOT NULL AUTO_INCREMENT,
+name VARCHAR(100) NOT NULL,
+type ENUM ('0','1') NOT NULL,
+email_address VARCHAR(200) NOT NULL,
+phone_number CHAR(13) NOT NULL,
+password VARCHAR(100) NOT NULL,
+PRIMARY KEY (id)
+);
+
+<!-- This is a query that creates the vendor_food table-->
+
+CREATE TABLE vendor_food 
+(
+id INT NOT NULL AUTO_INCREMENT,
+food_id INT NOT NULL,
+admin_id INT NOT NULL,
+name VARCHAR(100) NOT NULL,
```

Comment:
Some names are obviously longer than  100 characters, we could do 250 to be on a safer side